gladier_xpcs/tools/result_transfer.py

```diff
@@ -5,9 +5,29 @@ class ResultTransfer(GladierBaseTool):
 
     flow_definition = {
         'Comment': 'Transfer a file or directory in Globus',
-        'StartAt': 'ResultTransfer',
+        'StartAt': 'ResultTransferChoice',
         'States': {
-            'ResultTransfer': {
+            "ResultTransferChoice": {
+                "Comment": "Determine if the document should be cataloged in Globus Search",
```

Comment:
I think this could use more discussion. We currently publish all files to eagle, which makes two separate locations where data is being archived.

Making transfer optional here was mainly for testing, where the required path for "output" was not clear to me. Could you specify where data should go back to source, and I can test that?